es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -15,100 +15,111 @@
                 </div>
             </div>
         </div>  <!-- Make an impact -->
-        <div class="container text-gray font-size-75 pt-200 pb-300">
-            <div class="row">
-                <!-- Link columns -->
-                <div class="col col-12 col-lg-8 order-lg-1">
-                    <div class="row">
-                        <div
-                            v-for="column in content.columns"
-                            :key="column.title"
-                            class="col col-6 col-md-3 mb-150">
-                            <ul class="list-unstyled">
-                                <li class="font-weight-bolder font-size-100 mb-50">
-                                    {{ column.title }}
-                                </li>
-                                <li
-                                    v-for="link in column.links"
-                                    :key="link.text"
-                                    class="mt-25 mb-50">
-                                    <es-nav-bar-link
-                                        :href="link.url"
-                                        :target="link.newTab ? '_blank' : null"
-                                        class="text-reset">
-                                        {{ link.text }}
-                                    </es-nav-bar-link>
-                                </li>
-                            </ul>
+        <div class="bg-blue-800 py-200 px-200">
+            <div class="container px-0">
+                <div class="row">
+                    <!-- Link columns -->
+                    <div class="col col-12 col-lg-8 order-lg-1 text-white">
+                        <div class="row">
+                            <div
+                                v-for="column in content.columns"
+                                :key="column.title"
+                                class="col col-6 col-md-3 mb-150">
+                                <ul class="list-unstyled">
+                                    <li class="font-weight-bold font-size-75 font-size-md-100 mb-50">
```

Comment:
The Figma has 16px of spacing below category headers on mobile and (I think) 8px on desktop, so instead of `mb-50`, we should probably have `mb-100 mb-lg-50` here.